src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -106,10 +106,43 @@ internal partial class Interop
     [DllImport(LibgpiodLibrary, SetLastError = true)]
     internal static extern int GetGpiodLineValue(SafeLineHandle line);
 
+    /// <summary>
+    /// Check if line is no used (not set as Input or Output, not listening events).
+    /// </summary>
+    /// <param name="line">GPIO line handle</param>
+    /// <returns>0 or 1 if the operation succeeds. On error this routine returns -1 and sets the last error number.</returns>
```

Comment:
this returns comment is either wrong or the return type of the function is wrong (bool)

Reply:
Good catch, copy paste issue 😛